risc0/bigint2/src/lib.rs

```diff
@@ -12,9 +12,75 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#[cfg(feature = "num-bigint-dig")]
-extern crate num_bigint_dig as num_bigint;
-
 pub mod ec;
```

Comment:
```suggestion
#[cfg(features = "unstable)]
pub mod ec;
```
I think gating `ec`, `ffi` and `rsa` modules on the `unstable` feature will give you more flexibility to change things as more crates are patched.

Reply:
thank you!